alibi_detect/od/llr.py

```diff
@@ -144,6 +144,10 @@ def fit(self,
             Callbacks used during training.
         """
         input_shape = X.shape[1:]
+        optimizer = optimizer() if isinstance(optimizer, type) else optimizer
```

Comment:
This detector is a special case. The `optimizer` is called on two separate models during `fit`, which now leads to errors. To fix the `optimizer` is cloned so that separate optimizers are used for each model.